### ISSUE TITLE
[eslint-patch] error with eslint@7.12

### ISSUE THREAD

**Issue by kaiserfedor** [microsoft/rushstack#2325]
**Is this a feature or a bug?**

- [ ] Feature
- [x] Bug

**Please describe the actual behavior.**

`eslint-patch` does not work with eslint@7.12. If rollback to eslint@7.11 everything is fine. 

Error:
```
Cannot read config file: /Users/fedorkaiser/projects/monorepo/packages/app-candidate-v2-vk/node_modules/@worki/eslint-config/index.js
Error: Failed to patch ESLint because the calling module was not recognized
Referenced from: /Users/fedorkaiser/projects/monorepo/packages/app-candidate-v2-vk/package.json
```

**If the issue is a bug, how can we reproduce it? Please provide detailed steps and include a GitHub branch if applicable. Your issue will get resolved faster if you can make it easy to investigate.**

Install eslint@7.12 and run eslint.

**What is the expected behavior?**

eslint should run without error

**If this is a bug, please provide the tool version, Node.js version, and OS.**

* **Tool:** @rushstack/eslint-patch
* **Tool Version:** 1.0.5
* **Node Version:** 12.16.1
  * **Is this a LTS version?** yes
  * **Have you tested on a LTS version?** yes
* **OS:**  MacOS Mojave 10.14.6 

Thanks for your great job!

**Comment by octogonz** [comment]
Isn't this already fixed in `@rushstack/eslint-patch` version 1.0.6?

**Comment by kaiserfedor** [comment]
Yes, with 1.0.6 everything is ok.
Sorry for false alarm.
